src/@next/Menu/Menu.tsx

```diff
@@ -11,6 +11,7 @@ export interface Option {
   disabled?: boolean;
   id?: string;
   label: string | React.ReactNode;
+  sublabel?: string | React.ReactNode;
```

Comment:
string is a member of ReactNode type, so no need to re-state it.

```ts
    type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;
    type ReactFragment = {} | Iterable<ReactNode>;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

Reply:
Updated!